app/uk/gov/hmrc/lisaapi/controllers/AccountController.scala

```diff
@@ -95,7 +95,7 @@ class AccountController extends LisaController {
           auditService.audit(
             auditType = "accountNotCreated",
             path = getEndpointUrl(lisaManager),
-            auditData = creationRequest.toStringMap  + ("lisaManagerReferenceNumber" -> lisaManager) + ("reasonNotCreated" -> ErrorInvestorNotFound.errorCode)
+            auditData = creationRequest.toStringMap ++ Map("lisaManagerReferenceNumber" -> lisaManager, "reasonNotCreated" -> ErrorInvestorNotFound.errorCode)
```

Comment:
Can the ++Map() be split across multiple lines in all instances? Under create/transfer it's all on one line but under close it's split across multiple.